_episodes/09-pull-requests.md

```diff
@@ -133,17 +211,18 @@ $ git commit -m "Added file on france"
 {: .bash}
 
 ~~~
-[master 79a312a] Added file on france
+[addFrance 79a312a] Added file on france
  1 file changed, 2 insertions(+), 2 deletions(-)
 ~~~
 {: .output}
 
 Now we can push those changes to our forked version of the repo.
 Since we don't have permissions to the upstream repo or we want to suggest these changed be
```

Comment:
This sentence was somewhat confusing to me at first read, thought I think I get what is trying to be conveyed here. 

It could also be stated: "In the case that we don't have permission to push directly to the upstream repo or that we would like our changes to be reviewed regardless, we won't push directly to the upstream repository. Instead we will push the name of our new branch to the remote linked to our fork, origin."